### PR TITLE
Letter occurrences per word

### PR DESCRIPTION
**Pull request by NataliSam**

### CODE REVIEW

**Comment by ViktoriiaVolkova** [comment]
Checks were failed, fix it please

File: tasks/anyPrimeNumber/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { primeInRange } from "./index";
+
+describe("primeInRange", () => {
+  it(" Prime numbers 10, 15", () => {
+    expect(primeInRange(10, 15)).toBe(true);
```
Comment (ViktoriiaVolkova) [inline tasks/anyPrimeNumber/index.test.ts]: please use .toBeTruthy() instead of .toBe(true)

File: tasks/anyPrimeNumber/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  });
+});
+
+describe("primeInRange", () => {
+  it("Prime numbers 62, 66", () => {
+    expect(primeInRange(62, 66)).toBe(false);
```
Comment (ViktoriiaVolkova) [inline tasks/anyPrimeNumber/index.test.ts]: please use .toBeFalsy() instead of .toBe(false)

File: tasks/letterOccurrences/index.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+// 	return str
+// 		.toLowerCase()
+// 		.split(" ")
+// 		.reduce((acc, word) => {
+// 			return {
+// 				...acc,
+// 				[word]: [...word].filter(el => el === char.toLowerCase()).length
+// 			}
+// 		}, {})
+// }
```
Comment (ViktoriiaVolkova) [inline tasks/letterOccurrences/index.ts]: please remove comment

File: tasks/letterOccurrences/indes.test.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      an: 1,
+      apple: 1,
+      a: 1,
+      day: 1,
+      keeps: 0,
+      archeologist: 1,
+      "away...": 2,
+    });
+  });
+});
```
Comment (ViktoriiaVolkova) [inline tasks/letterOccurrences/indes.test.ts]: please update file name to index.test.ts

**Review by siarhei-papkou — APPROVED** [review]

**Pull request merged**